src/daemon/usb.c

```diff
@@ -215,6 +239,9 @@ static const devcmd* get_vtable(ushort vendor, ushort product){
 /// \n Should this function be declared as pthread_t* function, because of the defintion of pthread-create? But void* works also...
 ///
 static void* devmain(usbdevice* kb){
+    pthread_t onframe_thread;
+    pthread_create(&onframe_thread, NULL, onframe, kb);
```

Comment:
I check that on Windows but i think yes. I implemented a NodeJS implementation make by other person

Reply:
I check on windows with wireshark and yes is necessary because iCue send 

0000   1b 00 40 d4 88 42 0c ab ff ff 00 00 00 00 09 00   ..@..B..........
0010   00 01 00 02 00 01 01 40 00 00 00 33 ff 00 00 00   .......@...3....
0020   00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00   ................
0030   00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00   ................
0040   00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00   ................
0050   00 00 00 00 00 00 00 00 00 00 00                  ...........

Packet everytime

Reply:
And this packet corresponds to this in my code :

```c
void onframe_lighting_node(usbdevice* kb) {
    uchar pktend[MSG_SIZE] = {0x33, 0xff}; // Must be send everytime to keep modification
    if(!usbsend(kb, pktend, 1)) return;
}
```

Reply:
And this packet is send also after end to send color data. But, iCue send every time the color data to all devices so it not necessary for Windows. But, on ckb-next,  updatergb is not call everytime because is not necessary for devices excepts Lighting node. So i add a thread to send a packet every frame only on device where it is necessary.